inbm/cloudadapter-agent/cloudadapter/cloud/client/connections/mqtt_connection.py

```diff
@@ -89,6 +89,13 @@ def get_client_id(self) -> Optional[str]:
         """
         return self._client_id
 
+    def set_dispatcher_state(self, state: str) -> Optional[str]:
```

Comment:
it may be confusing to have this function do nothing--it would be better to either implement it, or raise a Not Implemented exception

Reply:
Not Implemented exception added.